### PR TITLE
[GSSOC'23] Created a template for Issues and Bugfixes

### PR DESCRIPTION
**Pull request by mkswagger**
## Description
Created and Added a new template for issues and bug fixes, to help the new contributors create an issue with the required details 

## Pull Request Checklist

Please make sure that your PR meets the following requirements:

- [X] The app is working correctly after applying the changes.
- [x] The app is responsive and displays properly on different screen sizes and devices.
- [x] The changes are thoroughly documented in the appropriate files or documentation sections.
- [x] The coding conventions and style guidelines are followed consistently.

## Checklist

Please check all applicable boxes and provide any additional information if needed.

- [x] I have tested the changes locally and ensured that the app is functioning as expected.
- [x] I have verified that the app is responsive and looks good on various screen sizes and devices.
- [x] I have reviewed the documentation and ensured that it accurately reflects the changes made.
- [x] I have followed the coding conventions and style guidelines defined in the project.
- [x] I have run the automated tests and all of them pass.
- [x] I have considered the performance implications of the changes made.
- [x] I have considered the security implications of the changes made.
- [x] I have checked for any potential regressions or side effects caused by the changes.
- [x] I have performed a self-review of my code.
- [x] Starred the repository.
- [x] Added My Name to the Contributor's List.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.

# Screenshots - If Any (Optional)

## Additional Notes(Optional)

Please provide any additional information or context that may be helpful for reviewing the pull request.

## Related Issues

This pull request is with reference to issue #176 

## Reviewers

@anmode  please check my pull request

### CODE REVIEW

**Comment by mkswagger** [comment]
And where to add my name in the contrubutors list, I cant find where to add

File: ISSUE_TEMPLATE.md
Comment (anmode) [inline ISSUE_TEMPLATE.md:1]: but issue template is there already If i am not wrong!

**Review by anmode — CHANGES_REQUESTED** [review]

**Review by anmode — COMMENTED** [review]
?

**Comment by anmode** [comment]
> And where to add my name in the contrubutors list, I cant find where to add

it added automatically when you PR got merged to codebase

File: ISSUE_TEMPLATE.md
Comment (mkswagger) [reply ISSUE_TEMPLATE.md:1]: im sorry I couldnt find it in the repo 
Is there any other issue u want me to work on?

**Pull request closed without merging**